Computer-SpaceGames/Starship-Takeoff/cpp/src/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.13)
+
+# set the project name
+project(my_project)
+
+# GoogleTest requires at least C++11
+set(CMAKE_CXX_STANDARD 11)
+
+# # add the executable
+add_executable(
+    Starship-Takeoff
+    main.cpp
+    game.cpp
+)
+
```

Comment:
There are some people we know who hate me for this but especially for a learning project I would always suggest adding `target_compile_options(Starship-Takeoff PRIVATE -Wall -Wextra -Werror)` here (and to all other C++ projects)